### PR TITLE
Qingwei/settings hide from virtual

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

File: src/_data/LiveData.js [hunk 1]
```diff
@@ -48,18 +48,22 @@ export const initUnauthorized = () => {
     getVideosFromPlayList().then(videos => api.events.emit('videos', videos));
 };
 
-export const initAuthorized = () => {
+export const initAuthorized = (authData) => {
+    const isVirtual = authData.authorize.loginid.startsWith('VR');
```
Comment (borisyankov) [inline src/_data/LiveData.js]: You don't need to put on top now. const & let are not hoisted, that is old school ;)

File: src/settings/SettingsCard.js [hunk 2]
```diff
@@ -14,20 +14,31 @@ export default class SettingsCard extends React.Component {
     static propTypes = {
 		actions: React.PropTypes.object.isRequired,
         settings: React.PropTypes.object.isRequired,
+		loginid: React.PropTypes.string.isRequired,
     };
 
 	render() {
-		const {settings, actions} = this.props;
+		const {loginid} = this.props;
+		const isVirtual = (loginid.startsWith('VRTC'));
```
Comment (borisyankov) [inline src/settings/SettingsCard.js]: extra commas for extra carefulness? :)

File: src/settings/SettingsCard.js [hunk 3]
```diff
@@ -14,20 +14,31 @@ export default class SettingsCard extends React.Component {
...
 			<Tabs
 				id="settings"
 				activeIndex={this.state.activeTab}
 				onChange={idx => this.setState({activeTab: idx})}
 				tabs={[
-					{text: 'Personal', component: <SettingsPersonalDetails settings={settings} actions={actions}/>},
-					{text: 'Security', component: <SettingsSecurity settings={settings} actions={actions}/>},
-					{text: 'Self Exclusion', component: <SettingsSelfExclusion settings={settings} actions={actions}/>},
-					{text: 'Limits', component: <SettingsLimits settings={settings} actions={actions}/>},
+					{text: 'Personal', component: <SettingsPersonalDetails {...this.props}/>},
```
Comment (borisyankov) [inline src/settings/SettingsCard.js]: You could have ?-ed the tabs var

File: src/settings/SettingsPersonalDetails.js [hunk 4]
```diff
@@ -7,15 +7,21 @@ export default class SettingsPersonalDetails extends React.Component {
 	static propTypes = {
 		actions: React.PropTypes.object.isRequired,
 		settings: React.PropTypes.object.isRequired,
+		loginid: React.PropTypes.string.isRequired,
 	};
 
 	render() {
-		const {settings, actions} = this.props;
+		const {settings, actions, loginid} = this.props;
+		const shouldHide = loginid.startsWith('VRTC');
```
Comment (borisyankov) [inline src/settings/SettingsPersonalDetails.js]: Rofl, even differing logic VR vs VRTC :0

File: src/settings/SettingsCard.js (hunk 2 above)
Comment (qingweibinary) [inline src/settings/SettingsCard.js]: extra parentheses * 
and I did something stupid elsewhere, and this is copied pasted from the stupid code, will fix

File: src/settings/SettingsCard.js (hunk 3 above)
Comment (qingweibinary) [inline src/settings/SettingsCard.js]: sorry, I dun get u?

File: src/settings/SettingsPersonalDetails.js (hunk 4 above)
Comment (qingweibinary) [inline src/settings/SettingsPersonalDetails.js]: will fix, is it ok to use VR at the moment ?

Comment (borisyankov) [inline src/settings/SettingsPersonalDetails.js]: May be use the full VRTC.

File: src/settings/SettingsCard.js (hunk 3 above)
Comment (borisyankov) [inline src/settings/SettingsCard.js]: const tabs = isVirtual ? [one list] : [another list];

then:

<Tabs tabs={tabs} />

Comment (qingweibinary) [inline src/settings/SettingsCard.js]: OK

**Pull request merged**